administrator/components/com_scheduler/src/Model/TasksModel.php

```diff
@@ -440,7 +440,7 @@ protected function populateState($ordering = 'a.title', $direction = 'ASC'): voi
     }
 
     /**
-     * Check if we have any enabled due tasks and no locked tasks.
+     * Check if we have any enabled due but no locked tasks.
```

Comment:
This comment, and subsequent changes, makes no sense. I am sure its just a "lost in translation" issue and I am happy to help but I just dont understand what it is trying to say

Reply:
Will revert the change. Thanks

```suggestion
     * Check if we have any enabled due tasks and no locked tasks.
```